server/routes/config/proposedAddress.js

```diff
@@ -45,6 +45,18 @@ module.exports = {
         licenceSection: 'curfewAddress',
         nextPath: '/hdc/proposedAddress/submit/'
     },
+
+    curfewAddressConfirmation: {
+        licenceSection: 'curfewAddress',
+        nextPath: null,
+        fields: [
+            {landLordHDCConsent: {}},
+            {hasElectricitySupply: {dependantOn: 'landLordHDCConsent', predicate: 'Yes'}},
+            {homeVisitConducted: {dependantOn: 'landLordHDCConsent', predicate: 'Yes'}},
+            {managedSafely: {}},
+            {managedSafelyReasons: {dependantOn: 'managedSafely', predicate: 'No'}}
+        ]
+    },
```

Comment:
I think this is in the wrong place. We're doing a route per 'journey' so I would have thought we would layout our config files in the same way

Reply:
think it needs to be in config/licenceConditions.js